docker/Dockerfile.devel

```
@@ -65,20 +65,36 @@ RUN CMAKE_VERSION=3.17 && \
     make -j"$(grep ^processor /proc/cpuinfo | wc -l)" install && \
     rm -rf /cmake-${CMAKE_BUILD}
 
-RUN pip install --force-reinstall --extra-index-url https://developer.download.nvidia.com/compute/redist/nightly nvidia-dali-nightly-cuda110
-
 WORKDIR /dali
 
+ARG DALI_DOWNLOAD_EXTRA_INDEX_URL=https://developer.download.nvidia.com/compute/redist/nightly
+ARG DALI_DOWNLOAD_PKG_NAME=nvidia-dali-nightly-cuda110
+ARG DALI_DOWNLOAD_VERSION
+ARG DALI_DOWNLOAD_EXTRA_OPTIONS
+
+RUN pip install --force-reinstall \
+    ${DALI_DOWNLOAD_EXTRA_OPTIONS} --extra-index-url ${DALI_DOWNLOAD_EXTRA_INDEX_URL} \
+    ${DALI_DOWNLOAD_PKG_NAME}`if [ -n "${DALI_DOWNLOAD_VERSION}" ]; then echo "==${DALI_DOWNLOAD_VERSION}"; fi`
```

```suggestion
    ${DALI_DOWNLOAD_PKG_NAME}${DALI_DOWNLOAD_VERSION:+==${DALI_DOWNLOAD_VERSION}}
```
But it may be less readable, just a thought.

---

Didn't know of this bash feature, thanks